test/index.ts

```diff
@@ -134,5 +134,9 @@ describe('XML formatter', function () {
     context('should collapse empty tags when forceSelfClosingEmptyTag=true', function () {
         assertFormat('test/data15/xml*-input.xml', { forceSelfClosingEmptyTag: true });
     });
+    
+    context('should not remove space with style before differently stylised word when prettifying xml', function () {
```

Comment:
@RobertLevett can you move the files from data16 to data12 folder? We already have the same configuration to format using the same option `{collapseContent: true}`

Reply:
Will do, just looking at test now too